sfa_dash/blueprints/delete.py

```diff
@@ -67,8 +67,26 @@ def post(self, uuid):
             # If the user was directed from anywhere other than
             # the confirmation page, redirect to confirm.
             return redirect(confirmation_url)
+        try:
+            self.metadata = self.api_handle.get_metadata(uuid)
+        except DataRequestException:
+            redirect_url = url_for(f'data_dashboard.{self.data_type}s')
+        else:
+            if self.metadata.get('site_id') is not None:
+                if self.data_type == 'site':
+                    redirect_url = url_for('data_dashboard.sites')
+                else:
+                    redirect_url = url_for(f'data_dashboard.{self.data_type}s',
+                                           site_id=self.metadata['site_id'])
+            else:
```

Comment:
only data_types that definitely have site_id or aggregate_id will get here? worth making this `elif 'aggregate_id' in self.metadata` and making `else` just the old redirect_url ?

Reply:
Good call, thanks.